opentelekomcloud/acceptance/cbr/resource_opentelekomcloud_cbr_vault_v3_test.go

```diff
@@ -263,6 +263,11 @@ resource "opentelekomcloud_cbr_vault_v3" "vault" {
     period_num    = 2
   }
```

Comment:
Initial ticket issue was that when tags were updated the resource had to be completely recreated.

So please add acceptance tests with at least 2 steps where different tags are applied. Tag checks are needed along with check that resource remains the same (this can be done by checking vault id).

It can be as simple as possible, we only have to check tags management in this test.

Reply:
Sorry, I am not very familiar with how I should write the corresponding test, can you help me?
And I think the update has actually been executed successfully, as verified in the previous call to UpdateResourceTags.